lib/std/http/method.zig

```diff
@@ -14,4 +14,87 @@ pub const Method = enum {
     OPTIONS,
     TRACE,
     PATCH,
+
+    /// Returns true if a request of this method is allowed to have a body
+    /// Returns null if it also depends on other conditions
+    pub fn requestHasBody(self: Method) ?bool {
+        return switch (self) {
+            .GET => false,
```

Comment:
`GET` can have a body.

Reply:
Huh I didn't know some methods couldn't have bodies.  Do you know where this is in the HTTP spec?

Reply:
its about *shouldn't*, something the client/server implementation can use in debug mode

Reply:
gonna rework the doc comment to reflect this